src/main/kotlin/ink/abb/pogo/scraper/Main.kt

```diff
@@ -93,7 +97,12 @@ fun main(args: Array<String>) {
     Log.normal("Logged in as $username with token ${auth.token.contents}")
 
     if (token.isBlank()) {
-        Log.normal("Set this token in your config to log in directly")
+        Log.normal("Setting this token in your config")
+        settings.setToken(auth.token.contents)
+        val out = FileOutputStream("config.properties")
+        settings.properties.store(out,null)
```

Comment:
Yeah, it's because `properties` reformulates all of the properties and print on the file, resetting it first.
The only solution I found on the web is using PropertiesConfiguration from [Apache Commons Configuration](http://commons.apache.org/proper/commons-configuration//userguide/howto_properties.html#Properties_files) but I haven't tried yet.